webpack.config.babel.js

```diff
@@ -35,7 +35,8 @@ let entryPoints = {
     markdown: ['./js/jquery/markdown.js'],
     mathjax: ['./js/jquery/compat/mathjax.js'],
     statistics: ['./js/jquery/statistics.js'],
-    fonts: ['./styles/partials/_fonts.scss']
+    fonts: ['./styles/partials/_fonts.scss'],
+    outdatedbrowser: ['./js/outdatedbrowser/index.js']
```

Comment:
since we just have a single file in `js/outdatebrowser/`, what about just making it `./js/outdatedbrowser.js`?